### PR TITLE
docs(components) add code doc comment on property.

### PR DESCRIPTION
**Pull request by LinBoLen**
All chinese doc almost come from the demo showcase. english is translate by me.
currently it include both english and chinese doc comment marked as two lines.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A
https://github.com/NG-ZORRO/ng-zorro-antd/issues/441

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by LinBoLen** [comment]
any feedback on this. please tell what I miss.

**Comment by vthinkxie** [comment]
will be reviewed later.

**Comment by LinBoLen** [comment]
no feedback yet. 
actually, I have noticed the comment written in code is like following, and there are no chinese language comment.
```typescript
// commment
@input foo: type;
```
I have added both english and chinese language comment. also, add `@type `@default` comment decorator.
really need feedback. so I can change the comment style. 

the semantic of the comment is important. but it can also be reviewed by the community.
the consistency of the comment is the first thing I want to take into consideration.

File: src/components/alert/nz-alert.component.ts [hunk 1]
```diff
@@ -58,13 +61,58 @@ import { FadeAnimation } from '../core/animation/fade-animations';
...
+
+  /**
+   * Whether use top banner
+   * 是否用作顶部公告
+   * @type {boolean}
+   */
   @Input() nzBanner = false;
+
+  /**
+   * Optional, default not display close button
```
Comment (tigercosmos) [inline src/components/alert/nz-alert.component.ts]: `Optional. Not display the close button by default.`
Please refer to it.

File: src/components/anchor/nz-anchor.component.ts [hunk 2]
```diff
@@ -66,10 +66,25 @@ export class NzAnchorComponent {
...
 
+  /**
+   * The bounds of anchor
+   * 锚点区域边界
+   * @type {number}
+   */
   @Input() nzBounds: number = 5;
 
+  /**
+   * the event of scroll, when scroll to a anchor
```
Comment (tigercosmos) [inline src/components/anchor/nz-anchor.component.ts]: `The event triggered by scrolling to an anchor`
`a anchor` -> `an anchor`
Please refer to it.

File: src/components/avatar/nz-avatar.component.ts [hunk 3]
```diff
@@ -31,14 +31,37 @@ export class NzAvatarComponent implements OnChanges {
...
+   * The shape of avatar `circle`   `square`
+   * 指定头像的形状 `circle`   `square`
+   * @type {string}
+   * @default circle
+   */
   @Input() nzShape: NzAvatarShape = 'circle';
 
+  /**
+   * 设置头像的大小
+   * @type {string} `default`   `large`   `small`
```
Comment (tigercosmos) [inline src/components/avatar/nz-avatar.component.ts]: No English here?

File: src/components/badge/nz-badge.component.ts [hunk 4]
```diff
@@ -88,13 +99,39 @@ export class NzBadgeComponent implements OnInit {
     return this._showZero;
   }
 
+  /**
+   * Don't show number, only there are red dot
```
Comment (tigercosmos) [inline src/components/badge/nz-badge.component.ts]: `there are only red dot`

File: src/components/button/nz-button.component.ts [hunk 5]
```diff
@@ -38,53 +38,103 @@ export class NzButtonComponent implements AfterContentInit {
...
+   */
   @Input()
   set nzLoading(value: boolean) {
     this._loading = value;
     this._setClassMap();
     this._setIconDisplay(value);
   }
 
+  /**
+   * Get whether loading
```
Comment (tigercosmos) [inline src/components/button/nz-button.component.ts]: `Get the loading status`

File: src/components/modal/nz-modal.component.ts [hunk 6]
```diff
@@ -137,31 +147,61 @@ export class NzModalComponent implements OnInit, OnDestroy, AfterViewInit {
     nzGlobalMonitor.setDocumentOverflowHidden(value);
   }
 
+  /**
+   * Set confirm loading
```
Comment (tigercosmos) [inline src/components/modal/nz-modal.component.ts]: `Set the button of "confirm loading"` may be more clear?

**Pull request closed without merging**